finagle-http2/src/test/scala/com/twitter/finagle/http2/Http2ListenerTest.scala

```diff
@@ -87,11 +87,12 @@ class Http2ListenerTest extends FunSuite {
                   |
                   |""".stripMargin.replaceAll("\n", "\r\n")))
 
-    assert(await(read()).get == """HTTP/1.0 400 Bad Request
-                                  |Connection: close
-                                  |Content-Length: 0
-                                  |
-                                  |""".stripMargin.replaceAll("\n", "\r\n"))
+    val responseLines = await(read()).get.lines.toIndexedSeq // not sure whether lines plays nice with JDK11, would only break running the *tests* anyway?
```

Comment:
why did we have to change this?  can we revert this change?  also this comment is hard to parse, is it still supposed to be here?

Reply:
+1

Reply:
This was changed because the order of the headers differs between 2.12 and 2.13. (Or at least differed when I was changing/fixing the tests.)

Reply:
reverted test completed successfully in 2.12 and 2.13, so reverted